### PR TITLE
chore: fixing api view that included unneeded values

### PR DESCRIPTION
**Pull request by alex-sheehan-edx**
Accidentally left in unneeded values from the api view

screenshot to verify that I got it right this time 🙃 
image
 

**Merge checklist:**
- [ ] Any new requirements are in the right place (do **not** manually modify the `requirements/*.txt` files)
    - `base.in` if needed in production but edx-platform doesn't install it
    - `test-master.in` if edx-platform pins it, with a matching version
    - `make upgrade && make requirements` have been run to regenerate requirements
- [ ] `make static` has been run to update webpack bundling if any static content was updated
- [ ] `./manage.py makemigrations` has been run
    - Checkout the [Database Migration](https://openedx.atlassian.net/wiki/spaces/AC/pages/23003228/Everything+About+Database+Migrations) Confluence page for helpful tips on creating migrations.
    - *Note*: This **must** be run if you modified any models.
      - It may or may not make a migration depending on exactly what you modified, but it should still be run.
    - This should be run from either a venv with all the lms/edx-enterprise requirements installed or if you checked out edx-enterprise into the src directory used by lms, you can run this command through an lms shell.
        - It would be `./manage.py lms makemigrations` in the shell.
- [ ] [Version](https://github.com/edx/edx-enterprise/blob/master/enterprise/__init__.py) bumped
- [ ] [Changelog](https://github.com/edx/edx-enterprise/blob/master/CHANGELOG.rst) record added
- [ ] Translations updated (see docs/internationalization.rst but also this isn't blocking for merge atm)

**Post merge:**
- [ ] Tag pushed and a new [version](https://github.com/edx/edx-enterprise/releases) released
    - *Note*: Assets will be added automatically. You just need to provide a tag (should match your version number) and title and description.
- [ ] After versioned build finishes in [GitHub Actions](https://github.com/edx/edx-enterprise/actions), verify version has been pushed to [PyPI](https://pypi.org/project/edx-enterprise/)
    - Each step in the release build has a condition flag that checks if the rest of the steps are done and if so will deploy to PyPi.
    (so basically once your build finishes, after maybe a minute you should see the new version in PyPi automatically (on refresh))
- [ ] PR created in [edx-platform](https://github.com/edx/edx-platform) to upgrade dependencies (including edx-enterprise)
    - This **must** be done after the version is visible in PyPi as `make upgrade` in edx-platform will look for the latest version in PyPi.
    - Note: the edx-enterprise constraint in edx-platform **must** also be bumped to the latest version in PyPi.

### CODE REVIEW

File: integrated_channels/canvas/models.py [hunk 1]
```diff
@@ -251,6 +251,13 @@ def _payload_data(self):
             subsection_name=self.subsection_name,
         )
 
+    @classmethod
+    def audit_type(cls):
```
Comment (alex-sheehan-edx) [inline integrated_channels/canvas/models.py:255]: to ensure this isn't picked up by the logging API queryset

File: integrated_channels/canvas/admin/__init__.py [hunk 2]
```diff
@@ -66,10 +66,10 @@ def customer_oauth_authorization_url(self, obj):
             return None
 
 
-@admin.register(CanvasLearnerAssessmentDataTransmissionAudit)
-class CanvasLearnerAssessmentDataTransmissionAuditAdmin(BaseLearnerDataTransmissionAuditAdmin):
+@admin.register(CanvasLearnerDataTransmissionAudit)
```
Comment (alex-sheehan-edx) [inline integrated_channels/canvas/admin/__init__.py:69]: 🤦

File: integrated_channels/integrated_channel/models.py [hunk 3]
```diff
@@ -373,13 +373,21 @@ def provider_id(self):
...
+    def get_completion_class_by_channel_code(cls, channel_code):
         """
-        Return the `LearnerDataTransmissionAudit` implementation for the particular channel_code, or None
+        Return the `LearnerDataTransmissionAudit` implementation related to completion reporting for
+        the particular channel_code, or None
         """
         app_label = channel_code_to_app_label(channel_code)
         for a_cls in cls.__subclasses__():
-            if a_cls._meta.app_label == app_label:
+            if a_cls._meta.app_label == app_label and a_cls.audit_type() == "completion":
```
Comment (alex-sheehan-edx) [inline integrated_channels/integrated_channel/models.py:390]: Make sure any filtered model is of type `completion`

**Review by kiram15 — APPROVED** [review]
Just fix the quality stuff!

**Pull request merged**